k8s/prod-network-uptime-cronjob.yaml

```diff
@@ -13,6 +13,12 @@ metadata:
   selfLink: /apis/batch/v1beta1/namespaces/default/cronjobs/airqo-predict-job
   uid: 9b71a693-43e2-4559-aff2-d821fc3bfe80
```

Comment:
@danielogen, forgive my ignorance... I see the same uuid " 9b71a693-43e2-4559-aff2-d821fc3bfe80 " being used in other YAML files. Is it okay???

Reply:
oh, thanks @NoahNsimbe for pointing that out. Actually, it does not matter - we can do without it.